### ISSUE TITLE
Failing CI on master

### ISSUE THREAD

**Issue by simonihmig** [ember-bootstrap/ember-bootstrap#1843]
CI on master is failing, due to accordion triggering an axe a11y failure. Idk how this change (axe update?) sneaked into master without failing CI first...

**Comment by simonihmig** [comment]
@jelhan unfortunately I don't have much time currently, would you be able to look into this? Or at least skip the test to unblock CI again?

**Comment by jelhan** [comment]
It is strange. The Axe rule does not even mention `role="tablist"`, which is flagged as invalid. And it is passing locally. :confused:

**Comment by jelhan** [comment]
It is failing for floating dependencies only. Got it failing locally as well. :+1:

**Comment by jelhan** [comment]
Per latest Bootstrap docs Accordion should not have a `role="tablist"`: https://getbootstrap.com/docs/5.3/components/accordion/ I will update accordingly.
